weekly-scan/template.yaml

```diff
@@ -64,20 +64,20 @@ objects:
                         stage('Scan the image') {
                             parallel (
                                 "RPM updates": {
-                                  sh (returnStdout: true, script: "docker run --rm --volumes-from `docker ps -qf name=${daemonset_name}` --entrypoint /bin/python ${image_name} /opt/scanning/yumupdates.py > yum-check-update 2>&1")
+                                  sh (returnStdout: true, script: "docker run --user root --rm --volumes-from `docker ps -qf name=${daemonset_name}` --entrypoint /usr/bin/python ${image_name_with_registry} /opt/scanning/yumupdates.py > yum-check-update 2>&1")
```

Comment:
notice that it is now referencing the correct image (tag) in scan, i.e. `image_name_with_registry`. This is the image we *pulled* in prior stage. Earlier we were referencing `image_name` here, which could actually be pulling that image.